scripts/linux/appimage/soh.sh

```diff
@@ -69,6 +117,14 @@ while [[ (! -e "$SHIP_HOME"/oot.otr) || (! -e "$SHIP_HOME"/oot-mq.otr) ]]; do
                         continue
                     fi
                     ;;
+                cfecfdc58d650e71a200c81f033de4e6d617a9f6)
```

Comment:
This hash is listed as supported in the readme but was missing in the linux/mac scripts. This could be considered a bug and probably back ported to Sulu as a bugfix.